ghub.py

```diff
@@ -610,13 +610,47 @@ def create_issue(issue_text=None):
         print("Sorry, something bad happened: " + str(result))
 
 
+def close_issue(number, issue_text=None):
+    """
+    Closes an opened issue.
+
+    PATCH /repos/:owner/:repo/issues/:number
+
+    """
+    (upstream_user, upstream_repo) = get_user_and_repo('upstream', 'origin')
+    if issue_text:
+        data = json.dumps({'body': issue_text}).encode('utf8')
+        url = GITHUB_API_URL + '/repos/%s/%s/issues/%d/comments' % (
+            upstream_user, upstream_repo, number)
+        result = make_github_request(
+            url, data, headers={'content-type': 'application/json'})
+        if 'body' in result:
+            pass
+        else:
+            print("Something bad happened: " + str(result))
+
+    data = json.dumps({'state': 'closed'}).encode('utf8')
+
+    url = GITHUB_API_URL + '/repos/%s/%s/issues/%d' % (
+        upstream_user, upstream_repo, number)
+
+    result = make_github_request(
+        url, data, method='PATCH', headers={'content-type': 'application/json'})
+
+    if 'title' in result:
+        print("Closed issue Number %d: Issue Tittle: %s" % (result['number'], result['title']))
```

Comment:
For consistency could this say something like:
```python
print("Closed issue #%d: %s" % (result['number'], result['title']))
```
to match create?

Reply:
Yes not a problem!